docs/source/install/install.rst

```diff
@@ -71,15 +64,14 @@ with no external dependencies.  However, some Python distributions
 (particularly on Windows and OS X), don't come with a working :mod:`readline`
 module.  The IPython shell will work without :mod:`readline`, but will lack
 many features that users depend on, such as tab completion and command line
-editing.  If you install IPython with :mod:`setuptools`, (e.g. with
-`pip`), then the appropriate :mod:`readline` for your platform will be
-installed.  See below for details of how to make sure you have a working
-:mod:`readline`.
+editing.  If you install IPython with :mod:`pip`,
+then the appropriate :mod:`readline` for your platform will be installed.
+See below for details of how to make sure you have a working :mod:`readline`.
 
 Installation using pip
 ----------------------
 
-If you have :mod:`setuptools` or :mod:`pip`, the easiest way of getting IPython is
+If you have :mod:`pip`, the easiest way of getting IPython is
 to simply use :command:`pip`:
```

Comment:
This now sounds a bit redundant: "If you have pip, the easiest way of getting IPython is to simply use pip"